docs/source/entities/quests.md

```diff
@@ -1 +1,20 @@
-# Quests
\ No newline at end of file
+# Quests
+
+Quests are a place to regroup all elements for a part of scenario or a scene.
+
+## Features
+Quests specific features are:
+* can be nested with "parent quest" field
+* a "date" field for reporting which will be displayed in side bar below pinned attributes
+* a "completed" field to keep track of ... completed quest in list view
+* calendar date entry: when filled up, it will automatically populate the calendar with reminder
+* an elements subpage
+
+## The elements subpage
+The Elements subpage allow you to collect all elements of a quest, being characters, locations, events, objects, notes, journals and so on.
+
+![Screenshot from 2023-04-18 09-43-21](https://user-images.githubusercontent.com/16939798/232707341-993dc66d-9cb0-479d-80ee-0b1dee52ad37.png)
+
+This way, you have at hand all usefull elements to keep track of your quest as well as subquests.
+Each reference has a "Role" field which all you to group and sorts elements in the Elements subpage.
+![quests](https://user-images.githubusercontent.com/16939798/232708216-f5c39959-dd0d-4583-873c-83189ae200dc.png)
```

Comment:
Same thing, please host the images in the git and not external to the docs

Reply:
done